imgui/src/widget/text.rs

```diff
@@ -36,14 +37,17 @@ impl<'ui> Ui<'ui> {
         style.end();
     }
     /// Renders text wrapped to the end of window (or column)
+    #[doc(alias = "TextWrapperd")]
```

Comment:
Probably "Wrapped" instead of "Wrapperd" although "Wrapperd" has some linguistic advantage -- what is a wrapper is not something where wrapperd's things